src/components/Button.tsx

```diff
@@ -11,19 +11,29 @@ interface ButtonProps {
     type?: 'button' | 'submit' | 'reset';
     width?: string;
     loading?: boolean;
+    isFormButton?: boolean;
```

Comment:
I would rename it to `size?: 'base' | 'small'`. In L25 it will be `size='base'` to give it a default value. Then use it in L36. This is better for semantic. @rayokamoto thoughts?

Reply:
yeah I think the intent would be clearer there as well.